### PR TITLE
#ATCP Added list call

### PR DESCRIPTION
**Pull request by ManojReddy-MSFT**
Added an example which shows how to retrieve the custom security attributes. We do not display the information by default and it's not intuitive so added it to the document

### CODE REVIEW

**Comment by Lauragra** [comment]
@ManojReddy-MSFT , can you please follow the Get started steps to set yourself up to be a contributor to the repo? https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211776/Get-started

File: concepts/custom-security-attributes-examples.md [hunk 1]
```diff
@@ -358,6 +358,51 @@ Content-type: application/json
 HTTP/1.1 204 No Content
 ```
 
+## List custom security attribute assignments
+
+### Example 1: List a single-valued custom security attribute assigned to a user
```
Comment (FaithOmbongi) [inline concepts/custom-security-attributes-examples.md:363]: "Get" over "List" for showing /users/{id}. List is more suitable for /users collection.
```suggestion
### Example 1: Get the custom security attributes assigned to a user
```

File: concepts/custom-security-attributes-examples.md [hunk 2]
```diff
@@ -358,6 +358,51 @@ Content-type: application/json
...
+
+The following example shows how to list a custom security attribute assignment assigned to the user.
```
Comment (FaithOmbongi) [inline concepts/custom-security-attributes-examples.md:365]:
```suggestion
The following example shows how to list a custom security attribute assignment assigned to the user. The custom security attribute is a multi-string value with the following settings:
```

File: concepts/custom-security-attributes-examples.md [hunk 3]
```diff
@@ -358,6 +358,51 @@ Content-type: application/json
...
+
+- Attribute set: `Engineering`
+- Attribute: `datacenter`
+- Attribute value: `Redmond`
```
Comment (FaithOmbongi) [inline concepts/custom-security-attributes-examples.md:369]: Let's show that this is a multi-string value
```suggestion
- Attribute data type: Collection of Strings
- Attribute value: ["Redmond"]
```

File: concepts/custom-security-attributes-examples.md [hunk 4]
```diff
@@ -358,6 +358,51 @@ Content-type: application/json
...
+
+#### Request
+
+
+<!-- {
+  "blockType": "request",
+  "name": "customsecurityattribute_remove_from_user_singlevalue"
```
Comment (FaithOmbongi) [inline concepts/custom-security-attributes-examples.md:376]:
```suggestion
  "name": "customsecurityattribute_get_for_user"
```

File: concepts/custom-security-attributes-examples.md [hunk 5]
```diff
@@ -373,7 +418,7 @@ The following example shows how to remove a custom security attribute assignment
 
 <!-- {
   "blockType": "request",
-  "name": "customsecurityattribute_remove_from_user_singlevalue"
+  "name": "customsecurityattribute__from_user_singlevalue"
```
Comment (FaithOmbongi) [inline concepts/custom-security-attributes-examples.md:421]: Maintain the example name for traceability
```suggestion
  "name": "customsecurityattribute_remove_from_user_singlevalue"
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Thank you @ManojReddy-MSFT for your proposed changes. I've left suggestions for minor fixes.

(Suggestion) We should also consider an example of LIST that applies $filter.

**Comment by ManojReddy-MSFT** [comment]
Thanks for the suggestions. I have committed them.  I will create another PR to add the filter example to this

**Review by FaithOmbongi — APPROVED** [review]
Thank you @ManojReddy-MSFT. Approved for merge.

**Pull request merged**